ros_comm/package.xml

```diff
@@ -1,6 +1,6 @@
 <package>
   <name>ros_comm</name>
-  <version>1.14.3</version>
+  <version>1.14.4</version>
```

Comment:
Unrelated change. Should be be reverted.

Reply:
Reverted.